src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -383,6 +383,7 @@ private void InitializeFilterList(UserSettings settings)
 
         private void PackageManagerLoaded(object sender, RoutedEventArgs e)
         {
+            Loaded -= PackageManagerLoaded;
```

Comment:
Does this change address some observable behavior or known bug, or are you just being conscientious?  I don't have an issue with it either way, but it stood out, being unrelated to gist of this PR.

Reply:
I removed this from the PR.